front/app/A.tsx

```diff
@@ -0,0 +1,11 @@
+import { ReactElement, useEffect, useState } from 'react';
+
+export default function A(): ReactElement {
```

Comment:
前回書いてsubmitし忘れてた件を一つ。
export defaultはimportが暗黙になるから基本的にはexportでそろえた方が設計的には推奨されている。

型としては`React.FunctionComponent<Props>`が一番いいね。以下のような形
```suggestion
interface Props {};

const A: React.FunctionComponent<Props> = () => (
```

---
余談として今回の場合はやらなくてもいいんだけどコンポーネントに関しては一番最後の行でexport A;とする方が一部のコンポーネントに対して色々する場合にも統一性があってよかったりする。

Reply:
おｋ。

ただDynamic Importはdefault exportじゃないとできなかった気が。

型については同意。（個人的にはコンポーネントなのは自明なのだから型書かなくてもよいという考えも持っているが）

default exportについては実装開始前に話そう

Reply:
おｋ。

ただDynamic Importはdefault exportじゃないとできなかった気が。

型については同意。（個人的にはコンポーネントなのは自明なのだから型書かなくてもよいという考えも持っているが）

default exportについては実装開始前に話そう